tests/devices/i22/test_dcm.py

```diff
@@ -171,6 +171,10 @@ async def test_reading(dcm: DoubleCrystalMonochromator):
     )
 
 
+@pytest.mark.skip(
+    reason="https://github.com/bluesky/ophyd-async/pull/594 assert_configuration() on numpy arrays is "
+    "broken"
+)
```

Comment:
Should: Is this still true? If so it needs a link to an open issue to fix/workaround